### DISCUSSION TITLE
Naming placeholder for ASIN (Amazon Standard Identification Number) book code

### DISCUSSION THREAD

**Discussion by MagratG** [audiamus/AaxAudioConverter#94] [Ideas]
Would it be possible to add the ASIN code (Agatha Raisin and a _**B0073X26SG**_LC_64_22050_Stereo) to the directory name to help with matching the decrypted files to the original .aax file please.

**Comment by audiamus** [comment]
I'm not sure. I wanted to make the output directory and file structure as human readable as possible. And I don't think the ASIN code contributes to that, particularly for multi-part books.

In the long term, with the option to directly access your Audible account, we will have a little local database that will store the state of all the books in your library and the destination folder of the plain audio, without the need for visible ASINs.

**Comment by MagratG** [comment]
If it was done as one of the placeholders and not included by default then I would have thought that the basic operation should still be clean.

**Comment by audiamus** [comment]
With multi-part books, each part file has its own ASIN, but the actual existence of parts is concealed in the plain audio output. So a simple placeholder won't suffice. Would require its own naming strategy. Handling all the different output structure combinations is hard enough to maintain already, I really don't want to make it even more complex. 

And with the future database and a list of all your books and their state, sortable by each column, should make the association easy, even without ASINs.

**Comment by MagratG** [comment]
OK, fair enough.  Thanks for listening to me.
